### ISSUE TITLE
Issue when use test_parser.parse_file("file")

### ISSUE THREAD

**Issue by enzofrnt** [anthonyharrison/lib4sbom#24]
Hi, all the information are here :
`File "/usr/lib/python3.11/site-packages/lib4sbom/cyclonedx/cyclonedx_parser.py", line 353, in _parse_component_xml
    self.cyclonedx_package.set_property(params["name"], params["value"])`

![image](https://github.com/anthonyharrison/lib4sbom/assets/63660254/43127a64-4255-4c16-a372-289261170e20)

**Comment by anthonyharrison** [comment]
@enzofrnt The problem is the way that syft defines a property. a property is a key/value pair and the value can be specified as an attribute 

```xml
            <properties>
                <property name="Foo" value="789"/>
                <property name="Bar" value="GHI"/>
            </properties>
```

or implicitly, like this

```xml
      <properties>
            <property name="Foo">789</property>
            <property name="Bar">GHI</property>
        </properties>
```

The code in the parser was only handling the explicit format (which is aligned with the examples on the CyclonedDX website). I will work on a fix to handle the implicit case

**Comment by anthonyharrison** [comment]
Fixed in release 0.6.0
